web/themes/custom/votegov/src/sass/components/columns.scss

```diff
@@ -4,6 +4,5 @@
 .vote-columns--two {
   @include at-media('tablet-lg') {
     column-count: 2;
-    column-gap: 2rem;
```

Comment:
This was messing with the wrapping of the arrow for the quick links.. and when i removed it it did not adjust much so didn't seem like it was needed... let me know if you feel other wise